### PR TITLE
Updated documentation link references

### PR DESCRIPTION
**Pull request by bdpedigo**
- [x] Does this PR have a descriptive title that could go in our release notes?
- [ ] Does this PR add any new dependencies?
- [ ] Does this PR modify any existing APIs?
   - [ ] Is the change to the API backwards compatible?
- [ ] Have you built the documentation (reference and/or tutorial) and verified the generated documentation is appropriate?

#### What does this implement/fix? Briefly explain your changes.
Just updates hyperlinks to docs in a few places

#### Any other comments?
Currently I believe the "latest" version of the docs is not working. I think this is the one we should point people to, but lets not merge this until "latest" is online

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,7 @@
 [![Paper shield](https://img.shields.io/badge/JMLR-Paper-red)](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
 [![PyPI version](https://img.shields.io/pypi/v/graspologic.svg)](https://pypi.org/project/graspologic/)
 [![Downloads shield](https://pepy.tech/badge/graspologic)](https://pepy.tech/project/graspologic)
-[![Docs shield](https://img.shields.io/readthedocs/graspologic)](https://graspologic.readthedocs.io/)
 ![graspologic CI](https://github.com/microsoft/graspologic/workflows/graspologic%20CI/badge.svg)
-[![codecov](https://codecov.io/gh/microsoft/graspologic/branch/dev/graph/badge.svg)](https://codecov.io/gh/microsoft/graspologic)
```
Comment (bdpedigo) [inline README.md:7]: I think this is not turned on currently?

File: README.md [hunk 2]
```diff
@@ -2,9 +2,7 @@
 [![Paper shield](https://img.shields.io/badge/JMLR-Paper-red)](http://www.jmlr.org/papers/volume20/19-490/19-490.pdf)
 [![PyPI version](https://img.shields.io/pypi/v/graspologic.svg)](https://pypi.org/project/graspologic/)
 [![Downloads shield](https://pepy.tech/badge/graspologic)](https://pepy.tech/project/graspologic)
-[![Docs shield](https://img.shields.io/readthedocs/graspologic)](https://graspologic.readthedocs.io/)
```
Comment (bdpedigo) [inline README.md:5]: no longer applies if we aren't using RTD

Comment (jonmclean) [reply README.md:5]: Good catch.

**Review by daxpryce — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (daxpryce) [reply README.md:7]: https://github.com/microsoft/graspologic/blob/dev/.github/workflows/build.yml#L86 I think it is?

Comment (daxpryce) [reply README.md:7]: I don't know how it actually works, but I know we're running code coverage as part of the build

Comment (daxpryce) [reply README.md:7]: And by "I know" I mean "I ... guess?"

Comment (bdpedigo) [reply README.md:7]: https://app.codecov.io/gh/microsoft/graspologic this looks not right to me

Comment (bdpedigo) [reply README.md:7]: possibly different from the check you mention though

Comment (bdpedigo) [reply README.md:7]: yeah i think the actual pytest cov thing is running, but this app that we used to use for reporting the results is not. at least that is my understanding

Comment (jonmclean) [reply README.md:7]: We do run code coverage but it does not seem to send the results to codecov.io.  The code coverage is so low that we look better by not reporting it :).

**Review by jonmclean — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jonmclean) [reply README.md:7]: Why oh why does this UI not refresh so I don't repeat what everyone else said? :)

**Pull request merged**